recipes/msys2/all/conanfile.py

```diff
@@ -1,21 +1,15 @@
 from conan import ConanFile
 from conan.errors import ConanInvalidConfiguration, ConanException
-from conan.tools.files import chdir, get, replace_in_file
+from conan.tools.files import chdir, get, replace_in_file, copy
+from conan.tools.layout import basic_layout
 import fnmatch
 import os
 import shutil
 import subprocess
 import errno
+import ctypes
 
-try:
-    import ctypes
-except ImportError:
-    pass
-except ValueError:
-    pass
-
-required_conan_version = ">=1.47.0"
-
+required_conan_version = ">=1.55.0"
```

Comment:
min conan version is 1.53.0 with these unecessary rm_safe, and without it's 1.47.0